packages/synapse-react-client/src/style/base/_core.scss

```diff
@@ -1147,6 +1135,24 @@ hr ~ .QueryWrapperPlotNav {
 
 .SynapseTable {
   min-height: 400px;
+  th {
+    background-color: map.get(SRC.$colors, 'gray-100');
+    border-right: 1px solid map.get(SRC.$colors, 'gray-400');
+  }
+  td,
+  th {
+    & > * {
+      padding: 8px 10px;
+    }
+  }
+
+  tr:nth-of-type(2n) {
+    background-color: #fbfbfc;
+  }
+
+  tbody > tr > td {
+    vertical-align: top;
+  }
```

Comment:
Styles to match designs / replicate removed bootstrap styles

Reply:
![image](https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/17580037/ff889404-8672-4a3a-b29f-2ffea2458df1)